frontend/public/style/_common.scss

```diff
@@ -99,14 +99,13 @@
   font-weight: normal;
 }
 
-.co-m-pane__name {
-  @include co-break-word;
-  align-items: baseline;
-  display: flex;
-  flex: 1;
-  margin-right: 10px;
-  min-width: 0; // necessary for wrapping since its a flex child
-}
+// .co-m-pane__name {
```

Comment:
Did you mean to leave this commented out?  If it's no longer needed, I recommend deleting it to avoid confusion.